connector/connect/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -170,6 +170,8 @@ message Expression {
 
   message Alias {
     Expression expr = 1;
-    string name = 2;
+    repeated string name = 2;
+    // Alias metadata expressed as a JSON map.
+    optional string metadata = 3;
```

Comment:
nit: previously @amaliujia added
```
  message DataSource {
    // Required. Supported formats include: parquet, orc, text, json, parquet, csv, avro.
    string format = 1;
    // Optional. If not set, Spark will infer the schema.
    string schema = 2;
    // The key is case insensitive.
    map<string, string> options = 3;
  }
```

Shall we also have a guideline for adding string-string map field? Do we prefer proto map field or just JSON string?

Reply:
That's a very good question. I think in this case it's not as easy possible because it actually depends on the type and is not a plain `map<string, string>`

Reply:
ah the metadata value can be any type, then JSON LGTM.